test/model/binding.dart

```diff
@@ -555,10 +555,13 @@ class FakeAndroidNotificationHostApi implements AndroidNotificationHostApi {
   }
   List<AndroidNotificationHostApiNotifyCall> _notifyCalls = [];
 
+  Iterable<StatusBarNotification> get activeNotifications => _activeNotifications.values;
+  final Map<(int, String?), StatusBarNotification> _activeNotifications = {};
+
   final Map<String, MessagingStyle?> _activeNotificationsMessagingStyle = {};
 
-  /// Clears all active notifications that have been created via [notify].
-  void clearActiveNotifications() {
+  /// Clears all [MessagingStyle] of active notifications that have been created via [notify].
+  void clearActiveNotificationsMessagingStyle() {
```

Comment:
`_activeNotificationsMessagingStyle` is a collection, so do we want to say `[MessagingStyle]s`, plural?